extensions/wikia/TrackingOptIn/js/trackingOptInProxy.js

```diff
@@ -0,0 +1,15 @@
+define('wikia.trackingOptIn', [
+	'wikia.window'
+], function (win) {
+
+	function init() {}
+
+	function pushToUserConsentQueue(callback) {
+		win.M.trackingQueue.push(callback);
+	}
+
+	return {
```

Comment:
Interface should also have `isOptedIn`:
https://github.com/Wikia/app/pull/15339/files#diff-c0d3a9f755350080de1d9f2bdca950fdR47